test/e2e/specs/splitting-merging.test.js

```diff
@@ -181,4 +181,34 @@ describe( 'splitting and merging blocks', () => {
 
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
+
+	it( 'should ensure always a default block', async () => {
```

Comment:
Probably as a follow up after this PR when dealing with this file we should rename the file and the description, as right now it is testing other behaviors like 'should delete an empty first line', and this test case that are not directly related with 'splitting and merging blocks'.